examples/decoding/plot_oasis_vbm_space_net.py

```diff
@@ -0,0 +1,65 @@
+"""
+Voxel-Based Morphometry on Oasis dataset with Space-Net prior
+=============================================================
+
+Predicting age from gray-matter concentration maps from OASIS dataset
+"""
+# Authors: DOHMATOB Elvis
+#          FRITSCH Virgile
+
+
+### Load Oasis dataset ########################################################
+import numpy as np
+from nilearn import datasets
+n_subjects = None  # more subjects requires more memory
+dataset_files = datasets.fetch_oasis_vbm(n_subjects=n_subjects)
+age = dataset_files.ext_vars['age'].astype(float)
+age = np.array(age)
+X = np.array(dataset_files.gray_matter_maps)
```

Comment:
I think that it would be good to use better names for 'X': some thing like 'gm_img' (applies to all related variables: X_train...).

Same thing for y: 'age' would be good.